src/Request/BasicResolver.php

```diff
@@ -58,10 +58,17 @@ protected static function getRequestHeaders(array $server)
 
         $headers = [];
 
-        foreach ($server as $name => $value) {
-            if (substr($name, 0, 5) == 'HTTP_') {
-                $headers[str_replace(' ', '-', ucwords(strtolower(str_replace('_', ' ', substr($name, 5)))))] = $value;
+        if (!function_exists('getallheaders')) {
+            foreach ($server as $name => $value) {
+                if (substr($name, 0, 5) == 'HTTP_') {
+                    $headers[str_replace(' ', '-', ucwords(strtolower(str_replace('_', ' ', substr($name, 5)))))] = $value;
+                }
+                else if ($name === "CONTENT_TYPE") {
```

Comment:
getallheaders does seem to be called on some of the test instances because before I put in 'fix failures on PHP 5.5.9' fix, most tests were successful except 5.5.9 - those were not setting Content-Type, so must have been using getallheaders.

Reply:
Fixed in 033bd8932af57e286b612d02b715b7efa598f323